linera-sdk/src/contract/exported_futures.rs

```diff
@@ -141,16 +141,22 @@ where
     Application: Contract + Send,
     Application::Error: 'static,
     Application::Storage: ContractStateStorage<Application> + Send + 'static,
+    Application::Effect: 'static,
```

Comment:
note to self: This can be cleaned up now (unless we don't include the constraint in the ContractAbi types)